stock_picking_batch_print_pickings/tests/test_batch_print_pickings.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo.exceptions import UserError
+from odoo.tests.common import TransactionCase
+
+
+class TestBatchPrintPickings(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.stock_location = cls.env.ref("stock.stock_location_stock")
+        cls.customer_location = cls.env.ref("stock.stock_location_customers")
+        cls.picking_type_out = cls.env["ir.model.data"]._xmlid_to_res_id(
```

Comment:
@EmilioPascual You can use `cls.env.ref("stock.picking_type_out")` instead

Reply:
Yes, yor're right. I have change it.
Thank you @rousseldenis .